client/state/happychat/selectors.js

```diff
@@ -40,8 +41,12 @@ export const HAPPYCHAT_CHAT_STATUS_PENDING = 'pending';
  */
 export const getGroups = ( state, siteId ) => {
 	const groups = [];
+	const siteDetails = getSite( state, siteId );
 
-	if ( isJetpackSite( state, siteId ) ) {
+	if ( isATEnabled( siteDetails ) ) {
+		// AT sites should go to WP.com even though they are jetpack also
+		groups.push( HAPPYCHAT_GROUP_WPCOM );
```

Comment:
> No need to repeat this if its already in the else. We could simplify with

I would rather not assume all `AT` sites are jetpack also, like I assumed all jetpack sites are only jetpack just to save a line of code.